scripts/docs_version_pragma_check.sh

```diff
@@ -32,13 +32,54 @@ SOLIDITY_BUILD_DIR=${SOLIDITY_BUILD_DIR:-build}
 source "${REPO_ROOT}/scripts/common.sh"
 source "${REPO_ROOT}/scripts/common_cmdline.sh"
 
+function versionGreater() {
+    v1=$1
+    v2=$2
+    ver1=( ${v1//./ } )
+    ver2=( ${v2//./ } )
+
+    if (( ${ver1[0]} > ${ver2[0]} )); then
+        return 0
+    elif (( ${ver1[0]} == ${ver2[0]} )) && (( ${ver1[1]} > ${ver2[1]} )); then
+        return 0
+    elif (( ${ver1[0]} == ${ver2[0]} )) && (( ${ver1[1]} == ${ver2[1]} )) && (( ${ver1[2]} > ${ver2[2]} )); then
+        return 0
+    fi
+    return 1
+}
+function versionEqual() {
```

Comment:
Why can't you use string comparison?

Reply:
Will update, can actually use simple string comparison